static/app/views/issueDetails/groupSimilarIssues/similarStackTrace/toolbar.tsx

```diff
@@ -15,6 +15,7 @@ import {trackAnalytics} from 'sentry/utils/analytics';
 type Props = {
   onMerge: () => void;
   groupId?: string;
+  itemsWouldGroup?: Array<Record<string, string | undefined>> | null;
```

Comment:
be more specific on the type
```suggestion
  itemsWouldGroup: Array<{id: string, shouldBeGrouped: boolean}> | null;
```

Reply:
Thanks for the feedback!